RecipeBook/ViewControllers/HomeScreenViewController/HomeScreenViewController.swift

```diff
@@ -18,8 +19,11 @@ class HomeScreenViewController: UIViewController {
         super.viewDidLoad()
 
         setupSearchButton()
+        let userDefaultsName = UserDefaultsService()
```

Comment:
Create a view model for this class and declare `User Defaults` as a property of that class. Create methods for getting and setting the values in the view model and call them from there.